src/workspace/resolver.rs

```diff
@@ -496,42 +631,23 @@ impl WorkspaceResolver {
   }
 }
 
-fn enhance_import_map_value_with_workspace_members<'a>(
-  mut import_map_value: serde_json::Value,
-  deno_jsons: impl Iterator<Item = &'a ConfigFile>,
-) -> serde_json::Value {
-  let maybe_imports = import_map_value
-    .as_object_mut()
-    .and_then(|o| o.remove("imports"));
-  let mut imports =
-    if let Some(serde_json::Value::Object(imports)) = maybe_imports {
-      imports
+fn normalize_export_name(sub_path: &str) -> Cow<str> {
```

Comment:
Oh yeah, can we make this public? https://github.com/denoland/deno/blob/v1.45.5/cli/lsp/jsr.rs#L270-L272

Reply:
Yup!